### ISSUE TITLE
content: CLS show incorrect "multiple unstable elements" example

### ISSUE THREAD

**Issue by MX66s** [GoogleChrome/web.dev#9329]
### What page(s) need to be updated?

* https://web.dev/cls/

### Why is this update needed?
In the last example that illustrates multiple unstable elements, The third unstable element have drawn red, dotted rectangles that covered only the "Zebra" item. It's should cover from the "Elephant" item to the end of "Zebra" (orange rectangle) as image below.

After union all three items it should cover 60% of viewport's area, so impact fraction should be 0.6

Then the layout shift score would be 0.6 * 0.3 = 0.18

![CLS - multiple unstable elements](https://user-images.githubusercontent.com/41833150/210358671-94ce4914-1c39-4fcf-8345-b1a9048fc266.png)

**Comment by tunetheweb** [comment]
@anniesullie can someone from the CSM team verify this? I tend to agree with the suggested change based on the explanations prior to this one, but am struggling to verify this with a [demo site I created](https://www.tunetheweb.com/experiments/cls-multiple/).

**Comment by mmocny** [comment]
Upon first looking at the screenshot, I thought that Crocodile->Zebra would be the boundaries of the shift.  Looking at @tunetheweb demo, that is what happens.

I also thought that what we would see is a shift of Dog, Horse, and Zebra, while the other elements are "new" and are not shifting.  That is what happens when I test.

I think that means the screenshot (while messy) is correct, and matches what I see locally.

Basically: 'Lion' and 'Tiger' are new elements that didn't actually shift, but because they push down Zebra, the area still counts.  But if you changed the demo to remove Zebra, I think you would not measure the bottom two rows as a shift.

**Comment by tunetheweb** [comment]
Chatted with @mmocny offline. He meant the screenshot in the issue is correct - not the one in the article.

Will update the doc.
